server/methods/core/cart.js

```diff
@@ -473,26 +473,31 @@ Meteor.methods({
 
     // if quantity lets convert to negative and increment
     const removeQuantity = Math.abs(quantity) * -1;
-    return Collections.Cart.update({
-      _id: cart._id,
-      items: cartItem
-    }, {
-      $inc: {
-        "items.quantity": removeQuantity
-      }
-    }, (error, result) => {
-      if (error) {
-        Logger.error(error);
-        Logger.error(Collections.Cart.simpleSchema().namedContext().invalidKeys(),
-          "error removing from cart");
-        return error;
-      }
-      if (result) {
-        Logger.info(`cart: removed variant ${
-          cartItem._id} quantity of ${quantity}`);
-        return result;
-      }
-    });
+    if (quantity >= cartItem.quantity) {
```

Comment:
We could also remove this conditional on 476 and add the check to line 450 above like:
```js
if (!quantity || quantity >= cartItem.quantity) {
```
...instead of re-calling the method with no quantity arg.  It accomplishes the same thing, but with a slightly easier to read logic flow.